src/OaiPmh/HarvesterFactory.php

```diff
@@ -50,6 +51,14 @@
  */
 class HarvesterFactory
 {
+    const PROXY_SETTINGS = [
```

Comment:
Code style requires a visibility here (`public` or `protected`, as you see fit).